salt/metalk8s/kubeadm/init/kubeconfig/kubelet.sls

```diff
@@ -2,10 +2,23 @@
 
 {%- set hostname = salt.network.get_hostname() %}
 
+{%- set ca_cert = salt['mine.get']('*', 'kubernetes_ca_server').values()[0] %}
```

Comment:
It's the opposite in terms of variable names: you have at first `ca_cert_b64` then `ca_cert` when you decode it ;-)

Reply:
hum, right, will change